library/core/src/main/java/com/google/android/exoplayer2/text/SubtitleDecoderFactory.java

```diff
@@ -83,6 +85,8 @@ public SubtitleDecoder createDecoder(Format format) {
         } else if (format.sampleMimeType.equals(MimeTypes.APPLICATION_CEA708)) {
           return clazz.asSubclass(SubtitleDecoder.class).getConstructor(Integer.TYPE)
               .newInstance(format.accessibilityChannel);
+        } else if (format.sampleMimeType.equals(MimeTypes.APPLICATION_DVBSUBS) && format.initializationData != null) {
+          return clazz.asSubclass(SubtitleDecoder.class).getConstructor(List.class).newInstance(format.initializationData);
```

Comment:
pro tip: You need to update library/core/src/main/proguard-rules.txt to avoid this failing when an app has been proguarded. Should be pretty self explanatory if you see what's done for 608/708 there.